bogota-july-2018/JSCalculator/assets/js/script.js

```diff
@@ -0,0 +1,81 @@
+
+var operations = {
+	"+": (num1, num2) => num1 + num2,
+	"-": (num1, num2) => num1 - num2,
+	"*": (num1, num2) => num1 * num2,
+	"/": (num1, num2) => num2 !== 0 ? num1/num2 : "Error!, use Clear button",
+	"":(num1, num2) => num2
+}
+
+var keyCodes = {
+	"106": "*",
+	"107": "+",
+	"109": "-",
+	"111": "/"
+}
+
+var res = 0;
```

Comment:
Var naming is important, `result` is an explicit name that helps other devs to understand your code.